sup3r/preprocessing/data_handling/dual_data_handling.py

```diff
@@ -19,7 +19,15 @@
 # pylint: disable=unsubscriptable-object
 class DualDataHandler(CacheHandlingMixIn, TrainingPrepMixIn):
     """Batch handling class for h5 data as high res (usually WTK) and netcdf
-    data as low res (usually ERA5)"""
+    data as low res (usually ERA5)
+
+    NOTE: When initializing the lr_handler it's important to pick a shape
```

Comment:
Nice, i'd like to see us add more of these "gotchas" directly to the docstrings

Reply:
agreed!